### PR TITLE
This library is integrated with Gradle and Maven

### PR DESCRIPTION
**Pull request by rafaelje**
This PR prepare the integration with gradle and maven with bintray.col, add Travis and add headers license for FunsionInventory and Fly-MDM

Ref: https://github.com/flyve-mdm/flyve-mdm-android-inventory/issues/7
Ref: https://github.com/flyve-mdm/flyve-mdm-android-inventory/issues/6
Ref: https://github.com/flyve-mdm/flyve-mdm-android-inventory/issues/2
Ref: https://github.com/flyve-mdm/flyve-mdm-android-inventory/issues/3

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.3"
+    defaultConfig {
+        applicationId "com.teclib.flyveinventorygrap"
```
Comment (ajsb85) [inline app/build.gradle]: Why com.teclib?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55" > "$ANDROID_HOME/licenses/android-sdk-license"
+  - echo -e "\n84831b9409646a918e30573bab4c9c91346d8abd" > "$ANDROID_HOME/licenses/android-sdk-preview-license"
+  - chmod +x gradlew
+  - ./gradlew dependencies || true
+
+before_script:
+  - echo no | android create avd --force -n test -t android-21 --abi armeabi-v7a
+  - emulator -avd test -no-skin -no-audio -no-window &
+  - android-wait-for-emulator
+  - adb shell input keyevent 82 &
```
Comment (ajsb85) [inline .travis.yml]: The library don't need the emulator

File: app/build.gradle (hunk 1 above)
Comment (ajsb85) [inline app/build.gradle]: This is only to test the library?

File: flyvemdminventory/build.gradle [hunk 3]
```diff
@@ -0,0 +1,120 @@
+apply plugin: 'com.android.library'
+
+ext {
+    bintrayRepo = 'inventory'
+    bintrayName = 'flyve-mdm-android-inventory'
+
+    publishedGroupId = 'com.flyvemdm.inventory'
+    libraryName = 'flyve-mdm-android-inventory'
+    artifact = 'flyvemdminventory'
```
Comment (ajsb85) [inline flyvemdminventory/build.gradle]: Artifact flyvemdminventory?

File: flyvemdminventory/build.gradle [hunk 4]
```diff
@@ -0,0 +1,120 @@
+apply plugin: 'com.android.library'
+
+ext {
+    bintrayRepo = 'inventory'
+    bintrayName = 'flyve-mdm-android-inventory'
+
+    publishedGroupId = 'com.flyvemdm.inventory'
```
Comment (ajsb85) [inline flyvemdminventory/build.gradle]: The Group ID no will be only `com.flyvemdm`?

File: flyvemdminventory/build.gradle [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    libraryName = 'flyve-mdm-android-inventory'
+    artifact = 'flyvemdminventory'
+
+    libraryDescription = 'A wrapper for FusionInventory on Android'
```
Comment (ajsb85) [inline flyvemdminventory/build.gradle]: Description: 

> Inventory client library written in Java for developing applications on Android

File: flyvemdminventory/build.gradle [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+    siteUrl = 'https://github.com/flyve-mdm/flyve-mdm-android-inventory'
```
Comment (ajsb85) [inline flyvemdminventory/build.gradle]: https://flyve-mdm.com

File: flyvemdminventory/src/main/java/com/flyvemdm/inventory/InventoryTask.java [hunk 7]
```diff
@@ -0,0 +1,198 @@
+package com.flyvemdm.inventory;
```
Comment (ajsb85) [inline flyvemdminventory/src/main/java/com/flyvemdm/inventory/InventoryTask.java]: Where is the license header?

File: app/build.gradle (hunk 1 above)
Comment (rafaelje) [inline app/build.gradle]: This is only for test

Comment (ajsb85) [inline app/build.gradle]: ok

File: .travis.yml (hunk 2 above)
Comment (rafaelje) [inline .travis.yml]: Removed

File: flyvemdminventory/build.gradle (hunk 4 above)
Comment (rafaelje) [inline flyvemdminventory/build.gradle]: flyvemdminventory is the old library will removed

File: flyvemdminventory/build.gradle (hunk 3 above)
Comment (rafaelje) [inline flyvemdminventory/build.gradle]: flyvemdminventory is the old library will removed

File: flyvemdminventory/build.gradle (hunk 5 above)
Comment (rafaelje) [inline flyvemdminventory/build.gradle]: flyvemdminventory is the old library will removed

File: flyvemdminventory/build.gradle (hunk 6 above)
Comment (rafaelje) [inline flyvemdminventory/build.gradle]: flyvemdminventory is the old library will removed

Comment (rafaelje) [inline flyvemdminventory/build.gradle]: I change on new one inventory

File: flyvemdminventory/src/main/java/com/flyvemdm/inventory/InventoryTask.java (hunk 7 above)
Comment (rafaelje) [inline flyvemdminventory/src/main/java/com/flyvemdm/inventory/InventoryTask.java]: In my next task

**Comment by ajsb85** [comment]
Where is the `pom.xml`?

**Comment by ajsb85** [comment]
Project Object Model or POM is the fundamental unit of work in Maven

**Comment by rafaelje** [comment]
The pow.xml file is generated dynamically by gradle with this 

`code:apply from: 'https://raw.githubusercontent.com/nuuneoi/JCenter/master/installv1.gradle'`

You can check the source here: https://raw.githubusercontent.com/nuuneoi/JCenter/master/installv1.gradle

**Comment by ajsb85** [comment]
Could you add the headers GPLv2 for the sources files from FusionInventory?
And add the headers of the GPLv3 for our sources.

**Comment by rafaelje** [comment]
@ajsb85 Yes I'll do it on my next task https://github.com/flyve-mdm/flyve-mdm-android-inventory/issues/3

**Pull request merged**